tests/test_pdf.py

```diff
@@ -5,6 +5,26 @@
 import json
 
 
+def test_hello_world_ctor():
```

Comment:
Maybe more explicit naming?
```suggestion
def test_minimum_model_spec():
```